go/exocom-mock/features/mock_reply.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Mock replies
+
+  Background:
+    Given an Exocom server
+    And I have mocked it to reply to "message.request" with "message.reply" and the payload:
```

Comment:
Let's please use plain English message names moving forward wherever possible. The dot notation is misleading, because it looks too much like RPC. In this case, the messages should be called `request message` and `reply message`.